src/client.js

```diff
@@ -535,11 +535,15 @@ const fileUploader = (client) => {
          * This is the exposed/public method for fileUploader instance which will do all the processing related to the upload process internally and returns the promise containing all the required data.
          * @ignore
          * @param file, where file is an instance of [File](https://developer.mozilla.org/en-US/docs/Web/API/File)
+         * @parma {"publishIfNeeded"|"none"|undefined} the post-processing action to execute. Defaults to "publishIfNeeded"
```

Comment:
typo `@parma`

Reply:
Thanks I've fixed this and included your suggestion of using a extensible "options" parameter